app/helpers/template_helper.rb

```diff
@@ -0,0 +1,14 @@
+  module TemplateHelper
+    def build_remediation_plan(remediation_ids, host)
+      entries = PreupgradeReportEntry.where(id: remediation_ids.split(','), host_id: host.id)
+      result = ""
+
+      entries&.each do |entry|
```

Comment:
In this case, `entries` will always be an array-like object

Reply:
yeah, no need for safe navigation here, it would mask the exception and would make it hard to debug if wrong parameters are sent

was there a specific reason for being safe here?